csrfguard/src/main/java/org/owasp/csrfguard/servlet/JavaScriptServlet.java

```diff
@@ -250,7 +260,7 @@ private static String parseDomain(final StringBuffer url) {
      * @return the first host in the list(e.g. "fox1" without port number). null if commaSeparatedHosts is invalid/null/blank
      */
     private static String getFirstHost(String commaSeparatedHosts) {
-        if (StringUtils.isBlank(commaSeparatedHosts)) { 
+        if (StringUtils.isBlank(commaSeparatedHosts)) {
             commaSeparatedHosts = null;
         }else {
```

Comment:
If you have made changes to the indentation, you could also add a space after the `}` in `}else`